tox.ini

```
@@ -2,7 +2,6 @@
 minversion = 4.4.4
 skip_missing_interpreters = {env:TOX_SKIP_MISSING_INTERPRETERS:True}
 envlist =
-    py37,
     py38,
     py39,
     py310,
```

Maybe add py311 as well, while we're here?

```suggestion
    py310,
    py311,
```